lnbits/extensions/satspay/migrations.py

```diff
@@ -18,8 +18,8 @@ async def m001_initial(db):
             completelink TEXT,
             completelinktext TEXT,
             time INTEGER,
-            amount INTEGER,
-            balance INTEGER DEFAULT 0,
+            amount {db.big_int},
```

Comment:
must use `f"""...`

Reply:
it fails with `sqlite3.OperationalError unrecognized token: "{"`. looks like the f is missing here.